content/html/reference/elements.md

```diff
@@ -0,0 +1,10 @@
+---
+title: HTML elements
+mdn_url: https://developer.mozilla.org/en-US/docs/HTML/Elements
+recipe: landing-page
+related_content: /content/related_content/html.yaml
+link_lists:
+  - title: Index of HTML elements
+    directory: "/content/html/reference/elements"
```

Comment:
I'm not sure if I'm nitpicking here but this line could be:
```
pages: "/content/html/reference/elements"
```
and the `build-json-*.js` scripts could worry about noticing that this is a directory and if so it basically replaces this, almost, like a macro, by `*.md` sorted alphabetically.

Reply:
Yes, that would work fine. I don't feel strongly either way about this. I think it's one of those places where you just choose between on explicit ("directory"/"pages") or implicit ("pages" and interpret it the right way).

Reply:
Been thinking more about it and I'm feeling more strongly about this one but I think it can wait because it's a bit out of context. A bit like HTML docs refers to images as either 
```html
<img src="../image.png"> or <img src="/images/foo.png">
```
but they don't do...
```html
<img src="/django_root/images/bar.png">
```

Speaking of images, some day we'll have images and then it becomes important that those images actually resolve when viewed in the GitHub UI for markdown preview I think. 

Long story short; some other day.

Reply:
Heck! I take back what I said. It actually reads, mentally, nicely like this.